test/app/favorites_test.dart

```diff
@@ -76,7 +77,7 @@ void main() {
       await pumpApp(tester, database);
       await tester.pumpAndSettle();
 
-      await tester.tap(find.byKey(Key('favoritesIcon')));
+      await tester.tap(find.byKey(favoritesIconKey));
```

Comment:
you can find by icon or tooltip as well...

Reply:
byIcon wouldn't be helpful since there is potentially more than one "heart" icon on the page at one time. Tooltip could work though